apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns-from-api.php

```diff
@@ -43,6 +50,10 @@ private function __construct() {
 			)
 		);
 
+		if ( ! get_theme_support( 'align-wide' ) ) {
+			$this->skip_full_width_patterns = true;
+		}
+
```

Comment:
I think this is good solution to the problem at hand!

Only that I'm wondering we could be more explicit in the var name, e.g., 

```
$this->supports_align_wide= false !== get_theme_support( 'align-wide' );

```

Not a big deal at all, just semantics and probably subjective at that :) Plus the fact that `can_register_pattern()` tells us all we need to know about "skipping".

And then 
```
	private function can_register_pattern( $pattern ) {
		if ( ! $this->has_align_wide_support && in_array( 'requires_align_wide', array_keys( $pattern['tags'] ), true ) ) {
			return false;
		}

....

```

Reply:
Nice, thanks for the feedback @ramonjd, I'll update the var name to be more specific, I like `$this->has_align_wide_support` 👍

Reply:
Abandoning using this variable name, as I've gone with the solution suggested below to infer the feature name from the tag slug.